README.md

```diff
@@ -14,6 +14,16 @@
     version: "latest"  # semver x.y.z
 ```
 
+### `dagger shell`
```

Comment:
We could make this the first option?

Reply:
maybe, i suspect `call` will still be the standard approach in ci configurations because of the named args